contracts/modules/ModuleStorage.sol

```diff
@@ -13,6 +13,8 @@ contract ModuleStorage {
 
     bytes32 public constant FEE_ADMIN = "FEE_ADMIN";
 
+    bytes32 public constant TREASURY = "TREASURY_WALLET";
```

Comment:
Please store the hash of "TREASURY_WALLET" and not "TREASURY_WALLET". All keys should be keccak hashes. Also, maybe declare it has internal? We don't need a getter for that in all the modules. It will be a waste of code size.